cfn.template.yaml

```diff
@@ -218,3 +224,89 @@ Resources:
             - '*'
       Roles:
         - !Ref deliveryRole
+
+  # ALB Solution
+  LambdaPermissionForALB:
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt Lambda.Arn
+      Action: lambda:InvokeFunction
+      Principal: elasticloadbalancing.amazonaws.com
+      SourceArn: !Ref TargetGroupForLambda
```

Comment:
As discussed IRL, this resource might need a DependsOn attribute to make sure it can be created from scratch in a new template.

Reply:
There's an example in here:

https://github.com/s0enke/cloudformation-templates/blob/master/templates/lambda-alb-sam-sample.yaml

Reply:
I am going to add this in a separate PR.